.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto eol=lf
```

Comment:
vscode stuffs ? 
github ci workflow ?

Reply:
https://stackoverflow.com/questions/2825428/why-should-i-use-core-autocrlf-true-in-git

Reply:
It was taken from the old repo, I assumed that iit is needed.